.github/workflows/goLint.yml

```diff
@@ -66,8 +70,17 @@ jobs:
         run: |
           git config --global url.https://${{ secrets.PAT }}@github.com/.insteadOf git+ssh://git@github.com
           git config --global url.git@github.com:.insteadOf https://github.com/
+      - name: Check that go.mod is tidied
+        if: inputs.skip-go-mod-tidy == false
```

Comment:
I'm not sure why this needs to be treated as a string, but actionlint complains if I use a ! to invert the boolean.